<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="24" passed="20">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="19626" started-at="2019-03-14T11:22:26Z" finished-at="2019-03-14T11:22:46Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="19626" started-at="2019-03-14T11:22:26Z" finished-at="2019-03-14T11:22:46Z">
      <class name="testCases.TS_001_Sales_Planning">
        <test-method status="PASS" signature="TC_001_verify_User_Able_TO_Navigate_Dashboard()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_001_verify_User_Able_TO_Navigate_Dashboard" duration-ms="5671" started-at="2019-03-14T16:52:26Z" finished-at="2019-03-14T16:52:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_verify_User_Able_TO_Navigate_Dashboard -->
        <test-method status="PASS" signature="TC_002_Verify_navigateTo_SalesPlanning()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_002_Verify_navigateTo_SalesPlanning" duration-ms="5564" started-at="2019-03-14T16:52:32Z" finished-at="2019-03-14T16:52:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_Verify_navigateTo_SalesPlanning -->
        <test-method status="PASS" signature="TC_003_Verify_District_Lable()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_003_Verify_District_Lable" duration-ms="803" started-at="2019-03-14T16:52:37Z" finished-at="2019-03-14T16:52:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_003_Verify_District_Lable -->
        <test-method status="PASS" signature="TC_004_Verify_TerriTory_Lable()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_004_Verify_TerriTory_Lable" duration-ms="49" started-at="2019-03-14T16:52:38Z" finished-at="2019-03-14T16:52:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_004_Verify_TerriTory_Lable -->
        <test-method status="PASS" signature="TC_005_Verify_GnDivision_Lable()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_005_Verify_GnDivision_Lable" duration-ms="55" started-at="2019-03-14T16:52:38Z" finished-at="2019-03-14T16:52:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_005_Verify_GnDivision_Lable -->
        <test-method status="PASS" signature="TC_006_Verify_StartDate_Lable()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_006_Verify_StartDate_Lable" duration-ms="63" started-at="2019-03-14T16:52:38Z" finished-at="2019-03-14T16:52:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_006_Verify_StartDate_Lable -->
        <test-method status="PASS" signature="TC_007_Verify_EndDate_Lable()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_007_Verify_EndDate_Lable" duration-ms="47" started-at="2019-03-14T16:52:38Z" finished-at="2019-03-14T16:52:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_007_Verify_EndDate_Lable -->
        <test-method status="PASS" signature="TC_008_Verify_Tower_Lable()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_008_Verify_Tower_Lable" duration-ms="46" started-at="2019-03-14T16:52:38Z" finished-at="2019-03-14T16:52:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_008_Verify_Tower_Lable -->
        <test-method status="PASS" signature="TC_009_Verify_Remarks_Lable()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_009_Verify_Remarks_Lable" duration-ms="47" started-at="2019-03-14T16:52:38Z" finished-at="2019-03-14T16:52:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_009_Verify_Remarks_Lable -->
        <test-method status="PASS" signature="TC_010_VerifyIs_District_DropdownEnabled()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_010_VerifyIs_District_DropdownEnabled" duration-ms="53" started-at="2019-03-14T16:52:38Z" finished-at="2019-03-14T16:52:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_010_VerifyIs_District_DropdownEnabled -->
        <test-method status="PASS" signature="TC_011_VerifyIs_Territory_DropdownEnabled()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_011_VerifyIs_Territory_DropdownEnabled" duration-ms="47" started-at="2019-03-14T16:52:38Z" finished-at="2019-03-14T16:52:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_011_VerifyIs_Territory_DropdownEnabled -->
        <test-method status="PASS" signature="TC_012_VerifyIs_GNDivisions_DropdownEnabled()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_012_VerifyIs_GNDivisions_DropdownEnabled" duration-ms="63" started-at="2019-03-14T16:52:38Z" finished-at="2019-03-14T16:52:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_012_VerifyIs_GNDivisions_DropdownEnabled -->
        <test-method status="PASS" signature="TC_013_VerifyIs_Towers_DropdownEnabled()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_013_VerifyIs_Towers_DropdownEnabled" duration-ms="71" started-at="2019-03-14T16:52:39Z" finished-at="2019-03-14T16:52:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_013_VerifyIs_Towers_DropdownEnabled -->
        <test-method status="FAIL" signature="TC_014_districtValue()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_014_districtValue" duration-ms="0" started-at="2019-03-14T16:52:39Z" finished-at="2019-03-14T16:52:39Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	Syntax error on token "for", delete this token
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	Syntax error on token "for", delete this token

	at pages.SalesPlanningPage.verifyDistrictDropdownValues(SalesPlanningPage.java:164)
	at testCases.TS_001_Sales_Planning.TC_014_districtValue(TS_001_Sales_Planning.java:213)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_014_districtValue -->
        <test-method status="PASS" signature="TC_014_select_districtValue()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_014_select_districtValue" duration-ms="1058" started-at="2019-03-14T16:52:39Z" finished-at="2019-03-14T16:52:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_014_select_districtValue -->
        <test-method status="FAIL" signature="TC_015_select_Terriotorry()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_015_select_Terriotorry" duration-ms="873" started-at="2019-03-14T16:52:40Z" finished-at="2019-03-14T16:52:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [COL - Colombo Depot] but found [COL - Colombo Depot
 ]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [COL - Colombo Depot] but found [COL - Colombo Depot
 ]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at proxy.verifications.verifyGlobalSelectDropDown(verifications.java:68)
	at pages.SalesPlanningPage.verifySelectedTerriotarry(SalesPlanningPage.java:178)
	at testCases.TS_001_Sales_Planning.TC_015_select_Terriotorry(TS_001_Sales_Planning.java:138)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_015_select_Terriotorry -->
        <test-method status="PASS" signature="TC_016_Verify_select_GN_Division()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_016_Verify_select_GN_Division" duration-ms="1970" started-at="2019-03-14T16:52:41Z" finished-at="2019-03-14T16:52:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_016_Verify_select_GN_Division -->
        <test-method status="PASS" signature="close()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="close" duration-ms="0" started-at="2019-03-14T16:52:43Z" finished-at="2019-03-14T16:52:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="PASS" signature="TC_017_verify_select_StarttDate()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_017_verify_select_StarttDate" duration-ms="515" started-at="2019-03-14T16:52:43Z" finished-at="2019-03-14T16:52:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_017_verify_select_StarttDate -->
        <test-method status="PASS" signature="TC_0181_verify_select_EndDate()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_0181_verify_select_EndDate" duration-ms="403" started-at="2019-03-14T16:52:43Z" finished-at="2019-03-14T16:52:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_0181_verify_select_EndDate -->
        <test-method status="FAIL" signature="TC_019_verify_Click_tower_Check_Box()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_019_verify_Click_tower_Check_Box" duration-ms="1042" started-at="2019-03-14T16:52:43Z" finished-at="2019-03-14T16:52:44Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at org.testng.Assert.assertFalse(Assert.java:73)
	at proxy.verifications.ClickonCheckBox(verifications.java:96)
	at pages.SalesPlanningPage.ClickonToweCheckBox(SalesPlanningPage.java:252)
	at testCases.TS_001_Sales_Planning.TC_019_verify_Click_tower_Check_Box(TS_001_Sales_Planning.java:171)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_019_verify_Click_tower_Check_Box -->
        <test-method status="PASS" signature="TC_020_verify_Confirm_message()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_020_verify_Confirm_message" duration-ms="717" started-at="2019-03-14T16:52:44Z" finished-at="2019-03-14T16:52:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_020_verify_Confirm_message -->
        <test-method status="FAIL" signature="TC_021_verify_Cancle_Submission()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_021_verify_Cancle_Submission" duration-ms="416" started-at="2019-03-14T16:52:45Z" finished-at="2019-03-14T16:52:46Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at org.testng.Assert.assertFalse(Assert.java:73)
	at pages.SalesPlanningPage.ClickOnCancleButton(SalesPlanningPage.java:288)
	at testCases.TS_001_Sales_Planning.TC_021_verify_Cancle_Submission(TS_001_Sales_Planning.java:194)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_021_verify_Cancle_Submission -->
        <test-method status="PASS" signature="TC_022_verify_SubmitSuccessFully()[pri:0, instance:testCases.TS_001_Sales_Planning@306e95ec]" name="TC_022_verify_SubmitSuccessFully" duration-ms="2" started-at="2019-03-14T16:52:46Z" finished-at="2019-03-14T16:52:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_022_verify_SubmitSuccessFully -->
      </class> <!-- testCases.TS_001_Sales_Planning -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
